### PR TITLE
Add a Population Ratio Metric

### PR DESCRIPTION
**Pull request by rebecca-burwei**
Not sure if I did it right

- Numerator listed first
- Following TOML standard: https://github.com/toml-lang/toml#example

### CODE REVIEW

File: definitions/firefox_desktop.toml [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+[metrics.ad_click_rate]
+friendly_name = "Ad Click Rate"
+description = """
+    This is a Population Ratio Metric. It is the total number of 
+    ad clicks over all clients in the population divided by the 
+    total number of SAP searches over all clients in the 
+    population.
+"""
+category = "search"
+depends_on = '["metrics.ad_clicks", "metrics.search_count"]'
```
Comment (scholtzan) [inline definitions/firefox_desktop.toml:12]:
```suggestion
depends_on = ["ad_clicks", "search_count"]
```

I was planning to update the docs today to add `depends_on`

File: jetstream/outcomes/firefox_desktop/firefox_suggest.toml [hunk 2]
```diff
@@ -8,3 +8,5 @@ description = "Number of searches performed in the urlbar, including search mode
 friendly_name = "urlbar searches"
 exposure_basis = ["exposures", "enrollments"]
 statistics = { deciles = {}, bootstrap_mean = {} }
+
+[metrics.ad_click_rate.statistics.bootstrap_mean]
```
Comment (scholtzan) [inline jetstream/outcomes/firefox_desktop/firefox_suggest.toml:12]:
```suggestion
[metrics.ad_click_rate.statistics.population_ratio]
numerator = "ad_clicks"
denominator = "search_count"
```

**Comment by scholtzan** [comment]
CI is currently failing due to a small bug. Working on a fix now

**Review by scholtzan — APPROVED** [review]

**Review by mikewilli — APPROVED** [review]
Hooray for new functionality! :tada:

**Pull request merged**